src/views/texts/TextDetails.vue

```diff
@@ -19,11 +19,22 @@
 								v-btn(color="primary" @click="addText")
 									v-icon(left) mdi-plus
 									| Add to project
+					v-alert(
+						type="warning"
+						dense
+						v-if="text.state !== 'OK'"
+					) The item is still pending with Giles. Wait until then for starting annotations
```

Comment:
Lets change GIles is pending to amphora is pending. Users probably wont know what Giles is but they will know what amphora is.